docs/references/react/use-organization.mdx

```diff
@@ -1,3 +1,126 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganization
+title: useOrganization()
+description: Access attributes of the currently active organization.
 ---
+
+# `useOrganization()`
+
+The `useOrganization()` hook gives you access to the current active organization attributes.
+
+```jsx
+type useOrganizationParams = {
+  invitationList: {
+    // Pagination params
+    limit?: number;
+    offset?: number;
+    },
+  membershipRequests: true
+    | (commonPaginationParams & {
+        status?: OrganizationInvitationStatus; // e.g. "pending"
+      });
+  memberships: true
+    | (commonPaginationParams & {
+        role?: MembershipRole[]; // e.g. ["basic_member", "admin"]
+      });
+  domains: true
+    | (commonPaginationParams & {
+        enrollmentMode?: OrganizationEnrollmentMode; // e.g. "automatic_invitation"
+      });
+};
+
+type commonPaginationParams = {
+  initialPage?: number;
+  pageSize?: number;
+  infinite?: boolean;
+  keepPreviousData?: boolean;
+}
```

Comment:
So we have a /types context. I'm thinking we can define types here, such as `useOrganizationParams`, `commonPaginationParams`, `OrganizationInvitationStatus`, `MembershipRole`, and `OrganizationEnrollmentMode`.

```suggestion
The `useOrganization()` hook gives you access to the current active organization attributes. This hook accepts [`useOrganizationParams`](/docs/refereces/javascript/types/use-organization-params).
```jsx
```